pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -184,7 +188,12 @@ func (s *ServiceImpl) GetNavTree(c *contextmodel.ReqContext, prefs *pref.Prefere
 		treeRoot.RemoveSectionByID(navtree.NavIDCfg)
 	}
 
-	if s.features.IsEnabled(c.Req.Context(), featuremgmt.FlagPinNavItems) && c.IsSignedIn {
+	evalCtx := openfeature.NewTargetlessEvaluationContext(map[string]any{})
+	enabled, err := s.openFeature.Client.BooleanValue(c.Req.Context(), featuremgmt.FlagPinNavItems, true, evalCtx)
```

Comment:
```suggestion
	enabled, err := s.openFeature.Client.BooleanValue(c.Req.Context(), featuremgmt.FlagPinNavItems, true, openfeature.TransactionContext(c.Req.Context()))
```

Reply:
actually, instead of using `BooleanValue`, best to use `Boolean` instead, then there's no need to handle the error (defaulting to the default value is good enough)

Reply:
Correct, but in this case the only thing we'd do to handle the error is default to something, which is what the error-less `Boolean()` method does.

If there's some other issue going on with the FF provider we'll see it elsewhere. In the general case where the static provider's being used, no errors _can_ happen, so it's sort of pointless...

Reply:
Thank you for explaining!